examples/systick/README.md

```diff
@@ -0,0 +1,96 @@
+examples/systick
+================
+
+This application runs 4 threads (tasks) using systick hardware timer for preemption
+* thread 'test0' every second
+* thread 'test1' every 2 seconds
+* thread 'test2' every 3 seconds
+* thread 'test3' never yield
+
+With no-preempt scheduling (classic RIOT way of life), thread 'test3' would block infinitely.
```

Comment:
Minor side-comment: RIOT's scheduler is preemptive (whenever there is an interrupt, the running thread is preempted and next the thread with highest priority is run), it just doesn't have ticks ;-).

Reply:
By default it is preemptive by implementation : no interrupt, no preemption. We could say it is a collaborative preemption.
With ticks, it is preemptive by nature.